heartbeat/mysql-common.sh

```diff
@@ -94,8 +96,12 @@ MYSQL_BINDIR=`dirname ${OCF_RESKEY_binary}`
 # Convenience variables
 
 MYSQL=$OCF_RESKEY_client_binary
+if [ "$OCF_RESKEY_replication_require_ssl" = "true" ]
```

Comment:
This should probably use shell function ocf_is_true() to handle uppercase/lowercase

Reply:
👍 Done.